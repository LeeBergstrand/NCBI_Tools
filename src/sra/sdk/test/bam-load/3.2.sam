@CO	Half aligned, half poorly aligned (bad position)
@CO	-----------------------------------------------------
@HD	VN:1.0	GO:none	SO:coordinate
@SQ	SN:20	LN:63025520	AS:NCBI37	
SRR006423.6228848	99	20	63025521	60	35M16S	=	60079	89	AGCTTGGAACCCTATAGAGTTGCTGAGTGCCAGGACCAGATCCTGGCCCTA	=;>@?;%=<>@AC@B79,>2C4B;7877%7%58:6%96.%8#$%//.%2:%	XC:i:35	XT:A:U	NM:i:0	SM:i:37	AM:i:37	X0:i:1	X1:i:0	XM:i:0	XO:i:0	XG:i:0	MD:Z:35	RG:Z:SRR006423	OQ:Z:IIIIII%I<EFIGD>-4&I(B*@++++)%)%'+0(%()&%+#$%&(&%&+&
SRR006423.6228848	147	20	60079	60	16S35M	=	60025	-89	GATCCTGTCAAAAAACAGGTGGTAAGGAAGGAGAGAGTGAAGGCACTGCCA	%."%/#:#9#""9%7$3<A%%A<#:@$@>AA>=57:<5=$<5)%4<?-:=;	XC:i:35	XT:A:U	NM:i:1	SM:i:37	AM:i:37	X0:i:1	X1:i:0	XM:i:1	XO:i:0	XG:i:0	MD:Z:27A7	RG:Z:SRR006423	OQ:Z:%&"%&#)#,#""+%($&(-%%/,#*-$2-22.-**-2,3&2+&%/II+III
