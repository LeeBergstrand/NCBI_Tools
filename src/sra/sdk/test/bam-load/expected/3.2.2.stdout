     ALIGNMENT_COUNT: 0, 1
          BASE_COUNT: 102
      BIO_BASE_COUNT: 102
      CMP_BASE_COUNT: 51
            CMP_READ: AGCTTGGAACCCTATAGAGTTGCTGAGTGCCAGGACCAGATCCTGGCCCTA
        COLOR_MATRIX: 0, 1, 2, 3, 4, 1, 0, 3, 2, 4, 2, 3, 0, 1, 4, 3, 2, 1, 0, 4, 4, 4, 4, 4, 4
              CSREAD: 323201020100233322210132122113012021012232021030023010312113020211222222020202031011021100000121120232
              CS_KEY: TT
           CS_NATIVE: false
      FIXED_SPOT_LEN: 0
         MAX_SPOT_ID: 1
         MIN_SPOT_ID: 1
                NAME: 1
            PLATFORM: SRA_PLATFORM_UNDEFINED
PRIMARY_ALIGNMENT_ID: 0, 1
             QUALITY: 30, 30, 30, 30, 30, 30, 1, 30, 20, 30, 30, 30, 30, 30, 20, 10, 10, 1, 30, 1, 30, 1, 30, 10, 10, 10, 10, 1, 1, 1, 1, 1, 10, 10, 1, 1, 1, 1, 1, 1, 10, 1, 1, 1, 1, 1, 1, 1, 1, 10, 1, 30, 30, 30, 10, 30, 30, 10, 1, 1, 10, 10, 1, 10, 10, 10, 10, 1, 1, 10, 10, 10, 10, 10, 10, 1, 10, 1, 1, 10, 10, 1, 1, 10, 1, 1, 1, 1, 1, 10, 1, 1, 1, 10, 1, 1, 1, 1, 1, 1, 1, 1
           RD_FILTER: SRA_READ_FILTER_PASS, SRA_READ_FILTER_PASS
                READ: AGCTTGGAACCCTATAGAGTTGCTGAGTGCCAGGACCAGATCCTGGCCCTATGGCAGTGCCTTCACTCTCTCCTTCCTTACCACCTGTTTTTTGACAGGATC
         READ_FILTER: SRA_READ_FILTER_PASS, SRA_READ_FILTER_PASS
            READ_LEN: 51, 51
            READ_SEG: [0, 51], [51, 51]
          READ_START: 0, 51
           READ_TYPE: SRA_READ_TYPE_BIOLOGICAL|SRA_READ_TYPE_FORWARD, SRA_READ_TYPE_BIOLOGICAL|SRA_READ_TYPE_REVERSE
          SIGNAL_LEN: 0
          SPOT_COUNT: 1
          SPOT_GROUP: SRR006423
             SPOT_ID: 1
            SPOT_LEN: 102
          TMP_KEY_ID: 0
            TRIM_LEN: 102
          TRIM_START: 0

================================== PRIMARY_ALIGNMENT ==================================
            ALIGN_GROUP: 
               ALIGN_ID: 1
             BASE_COUNT: 51
         BIO_BASE_COUNT: 51
             CIGAR_LONG: 16S27=1X7=
         CIGAR_LONG_LEN: 10
            CIGAR_SHORT: 16S35M
        CIGAR_SHORT_LEN: 6
     CLIPPED_CIGAR_LONG: 27=1X7=
 CLIPPED_CIGAR_LONG_LEN: 7
    CLIPPED_CIGAR_SHORT: 35M
CLIPPED_CIGAR_SHORT_LEN: 3
   CLIPPED_HAS_MISMATCH: 00000000000000000000000000010000000
 CLIPPED_HAS_REF_OFFSET: 00000000000000000000000000000000000
       CLIPPED_MISMATCH: C
        CLIPPED_QUALITY: 1, 1, 10, 1, 1, 10, 10, 1, 1, 10, 1, 10, 10, 10, 10, 10, 10, 1, 1, 10, 10, 10, 10, 1, 10, 10, 1, 1, 10, 30, 30, 10, 30, 30, 30
           CLIPPED_READ: AGGTGGTAAGGAAGGAGAGAGTGAAGGCACTGCCA
     CLIPPED_REF_OFFSET: 
         CMP_BASE_COUNT: 51
           COLOR_MATRIX: 0, 1, 2, 3, 4, 1, 0, 3, 2, 4, 2, 3, 0, 1, 4, 3, 2, 1, 0, 4, 4, 4, 4, 4, 4
                 CS_KEY: T
              CS_NATIVE: false
          EDIT_DISTANCE: 1
       GLOBAL_REF_START: 60078
           HAS_MISMATCH: 111111111111111100000000000000000000000000010000000
         HAS_REF_OFFSET: 100000000000000000000000000000000000000000000000000
                  LABEL: ploidy1
              LABEL_LEN: 7
            LABEL_START: 0
         LEFT_SOFT_CLIP: 16
                   MAPQ: 60
          MATE_ALIGN_ID: 
        MATE_CIGAR_LONG: 
    MATE_CIGAR_LONG_LEN: 
       MATE_CIGAR_SHORT: 
   MATE_CIGAR_SHORT_LEN: 
     MATE_EDIT_DISTANCE: 
            MATE_REF_ID: 
           MATE_REF_LEN: 
          MATE_REF_NAME: 
   MATE_REF_ORIENTATION: 
           MATE_REF_POS: 
        MATE_REF_SEQ_ID: 
            MAX_SPOT_ID: 1
            MIN_SPOT_ID: 1
               MISMATCH: GATCCTGTCAAAAAACC
          MISMATCH_READ: GATCCTGTCAAAAAAC===========================C=======
                   NAME: 1
               PLATFORM: SRA_PLATFORM_UNDEFINED
                QUALITY: 1, 1, 1, 1, 1, 1, 1, 1, 10, 1, 1, 1, 10, 1, 1, 1, 1, 1, 10, 1, 1, 10, 10, 1, 1, 10, 1, 10, 10, 10, 10, 10, 10, 1, 1, 10, 10, 10, 10, 1, 10, 10, 1, 1, 10, 30, 30, 10, 30, 30, 30
               RAW_READ: TGGCAGTGCCTTCACTCTCTCCTTCCTTACCACCTGTTTTTTGACAGGATC
              RD_FILTER: SRA_READ_FILTER_PASS
                   READ: GATCCTGTCAAAAAACAGGTGGTAAGGAAGGAGAGAGTGAAGGCACTGCCA
            READ_FILTER: SRA_READ_FILTER_PASS
               READ_LEN: 51
             READ_START: 0
              READ_TYPE: SRA_READ_TYPE_BIOLOGICAL|SRA_READ_TYPE_FORWARD
             REF_DELETE: 00000000000000000000000000000000000
                 REF_ID: 13
             REF_INSERT: 00000000000000000000000000000000000
                REF_LEN: 35
           REF_MISMATCH: 00000000000000000000000000010000000
               REF_NAME: 20
             REF_OFFSET: -16
        REF_ORIENTATION: true
                REF_POS: 60078
               REF_READ: AGGTGGTAAGGAAGGAGAGAGTGAAGGAACTGCCA
             REF_SEQ_ID: CM000682.1
              REF_START: 78
              REF_TABLE: REFERENCE
        RIGHT_SOFT_CLIP: 0
              SAM_FLAGS: 153
            SAM_QUALITY: """"""""+"""+"""""+""++""+"++++++""++++"++""+??+???
               SEQ_NAME: 1
            SEQ_READ_ID: 2
         SEQ_SPOT_GROUP: SRR006423
            SEQ_SPOT_ID: 1
             SPOT_COUNT: 1
             SPOT_GROUP: SRR006423
                SPOT_ID: 1
               SPOT_LEN: 51
           TEMPLATE_LEN: 0
             TMP_KEY_ID: 0, 0
               TRIM_LEN: 51
             TRIM_START: 0

